packages/aws-client/src/SNS.ts

```diff
@@ -4,10 +4,62 @@
 
 import pRetry from 'p-retry';
 import Logger from '@cumulus/logger';
+import {
+  PublishCommand,
+  SubscribeCommand,
+  UnsubscribeCommand,
+  ListSubscriptionsByTopicCommand,
+  CreateTopicCommand,
+  DeleteTopicCommand,
+  ConfirmSubscriptionCommand,
+} from '@aws-sdk/client-sns';
 import { sns } from './services';
 
 const log = new Logger({ sender: 'aws-client/sns' });
 
+/**
+ * Helper function for sending SNS messages for sdk v3, decides by type
+ *
+ * @param {any} messageParams - SNS message
+ * @param {string} messageType - Type of sns request
+ * @returns {Promise<any>}
+ */
+export const sendSNSMessage = async function (
+  messageParams: any,
```

Comment:
@Nnaga1  We shouldn't generally be making use of the <any> type *particularly* for the return (the interpreter has an ... interesting result here currently)  ,  IEach of the command objects have a defined type for input.

As doing part of this, should also consider removing the switch statement and consider using a generator function/TS generic - something like (this is an example of what I mean, not an implementation that will necessarily work):

```
interface GenericClass<T> {
  new (...args: unknown[]): T;
}
function returnSnsInstance<Type>(Arg: GenericClass<Type>, ...args: any[]): Type {
  return new Arg(...args);
}
```

The goal being to not break the typing around messageParams, as well as not allow an `any` return from this method for anything calling it via typescript. 

If the complexity of generalizing for all commands isn't worth the hassle because of the SDK's internal typing consider exporting a named method for each over using a switch as that will  likely be much cleaner.